src/applications/disability-benefits/526EZ/pages/privateMedicalRecordRelease.js

```diff
@@ -0,0 +1,171 @@
+      'ui:description': recordReleaseDescription,
+      'ui:title': disabilityNameTitle,
+      'view:limitedConsent': {
+        'ui:title': limitedConsentTitle,
+      },
+      limitedConsent: {
+        'ui:title': limitedConsentTextTitle,
+        'ui:options': {
+          expandUnder: 'view:limitedConsent',
+          expandUnderCondition: true,
+        },
+        'ui:required': (formData, index) =>
+          _.get(`disabilities.${index}.view:limitedConsent`, formData),
+      },
+      'view:privateRecordsChoiceHelp': {
+        'ui:description': limitedConsentDescription,
+      },
+      providerFacility: {
+        'ui:options': {
+          itemName: 'Provider',
+          viewField: PrivateProviderTreatmentView,
+          hideTitle: true,
+        },
+        items: {
+          providerFacilityName: {
+            'ui:title': 'Name of private provider or hospital',
+          },
+          'ui:validations': [validateDate],
+          treatmentDateRange: dateRangeUI(
+            'Approximate date of first treatment',
+            'Approximate date of last treatment',
+            'End of treatment must be after start of treatment',
+          ),
+          providerFacilityAddress: merge(addressUI('', false), {
+            street2: {
+              'ui:title': 'Street 2',
+            },
+            postalCode: {
+              'ui:title': 'Postal Code',
+              'ui:options': {
+                widgetClassNames: 'usa-input-medium',
+              },
+              'ui:validations': [
+                {
+                  validator: validateZIP,
+                },
+              ],
+            },
+          }),
+        },
+      },
+    },
+  },
+};
+
+export const schema = {
+  type: 'object',
+  properties: {
+    disabilities: {
+      type: 'array',
+      items: {
+        type: 'object',
+        required: ['providerFacility'],
+        properties: {
+          providerFacility: {
+            type: 'array',
+            minItems: 1,
+            maxItems: 100,
+            items: {
+              type: 'object',
+              required: ['providerFacilityName', 'providerFacilityAddress'],
+              properties: {
+                providerFacilityName: {
+                  type: 'string',
+                  minLength: 1,
+                  maxLength: 100,
+                },
+                treatmentDateRange: {
+                  type: 'object',
+                  properties: {
+                    from: {
+                      $ref: '#/definitions/date',
+                    },
+                    to: {
+                      $ref: '#/definitions/date',
+                    },
+                  },
+                  required: ['from', 'to'],
+                },
+                providerFacilityAddress: {
+                  type: 'object',
+                  required: [
+                    'street',
+                    'city',
+                    'country',
+                    'state',
```

Comment:
I think `addressUi` takes care of conditionally requiring `state` so that it's only required for countries that have a states list. I'd probably lean toward relying on that instead of making `state` always required in case there's a country that has none, unless your requirements say you have to make it required :)

Reply:
Removed `state` and `postalCode`